apps/wallet/tailwind.config.ts

```diff
@@ -30,7 +30,7 @@ export default {
 				7.5: '1.875rem',
 				8: '2rem',
 				15: '3.75rem',
-				'popup-height': '680px',
+				'popup-height': '600px',
```

Comment:
Is this intentional?

Reply:
Yes the popup seems to have a max height of 600px and just reverted the change 680px so it won't be broken in the popup view (related discussion [here](https://mysten-labs.slack.com/archives/C042YF60NH2/p1691416481268339))